1-js-and-async-programming/1-javascript-fundamentals/1-time-difference.js

```diff
@@ -16,6 +16,17 @@ timeDifference(7200, 3400); // Expected output: "01:03:20"
 
 */
 
-const timeDifference = (a, b) => {};
+const formatTime = (num) => {
+  num = Math.floor(num);
```

Comment:
It's better to keep your parameters without changes, so you can continue using them later.
```suggestion
  const numRoundedDown = Math.floor(num);
```